### PR TITLE
Rewards employee view

### PR DESCRIPTION
**Pull request by kajawitek**
Hi Nerds!

1. List of rewards for employee added. New reward specs added.
2. New method of logging in to tests added.
3. Reward column name changed from content to description.

### CODE REVIEW

File: spec/features/reward_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  let!(:employee) { create(:employee) }
+
+  before do
+    login_as employee, scope: :employee
+    visit root_path
+  end
+
+  it 'index and show rewards for employee user' do
+    click_link 'Rewards'
+    expect(page).to have_content 'Rewards Title Description Price'
```
Comment (ciejj) [inline spec/features/reward_spec.rb:14]: what are you checking here exactly? for me it looks like the headers of the Rewards index table. I think we should be checking if `reward.title`, `reward.price` and `reward.description` are being displayed here

File: spec/features/reward_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    click_link 'Show'
+    expect(page).to have_content 'Rewards Title: factory bot reward title 2 Description: Reward - description Back'
```
Comment (ciejj) [inline spec/features/reward_spec.rb:17]: Same as above - this looks weird you should be looking for values of reward object, not hardcoded ones

File: app/controllers/rewards_controller.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    @rewards = Reward.all
+  end
+
+  def show
+    @reward = Reward.find(params[:id])
+  end
+
+  private
+
+  def reward_params
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/controllers/rewards_controller.rb:14]: it's unnecessary -> deleted

File: app/views/rewards/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Title:</strong>
```
[the remark this answers is not in the document]
Comment (kajawitek) [reply app/views/rewards/show.html.erb:4]: Price added :)

File: spec/features/reward_spec.rb (hunk 1 above)
Comment (kajawitek) [reply spec/features/reward_spec.rb:14]: yes, we should check `reward.title`, `reward.price` and `reward.description` -> corrected

File: spec/features/reward_spec.rb (hunk 2 above)
Comment (kajawitek) [reply spec/features/reward_spec.rb:17]: corrected :)

**Pull request merged**